src/pages/en/guides/deploy/github.md

```diff
@@ -22,48 +22,51 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
 2. Create a new file in your project at `.github/workflows/deploy.yml` and paste in the YAML below.
 
     ```yaml
-    name: Github Pages Astro CI
+    name: Build astro site
 
     on:
-      # Trigger the workflow every time you push to the `main` branch
-      # Using a different branch name? Replace `main` with your branch’s name
       push:
-        branches: [main]
-      # Allows you to run this workflow manually from the Actions tab on GitHub.
-      workflow_dispatch:
```

Comment:
Is this still possible or not?

Reply:
(Meaning workflow_dispatch or is it not a real action anymore?)

Reply:
still possible! sorry I should've double checked how the old action looked. I kinda just replaced it with the workflow my team made!